CHANGELOG.md

```diff
@@ -1,5 +1,93 @@
 # Changelog
 
+## v0.31.0
+
+*March 16th, 2019*
+
+Special thanks to external contributors on this release:
+@danil-lashin, @guagualvcha, @siburu, @silasdavis, @srmo, @Stumble, @svenstaro
+
+This release is primarily about the new pubsub implementation, dubbed `pubsub 2.0`, and related changes,
+like configurable limits on the number of active RPC subscriptions at a time (`max_subscription_clients`).
+Pubsub 2.0 is an improved version of the older pubsub that is non-blocking and has a nicer API.
+Note the improved pubsub API also resulted in some improvements to the HTTPClient interface and the API for WebSocket subscriptions.
+This release also adds a configurable limit to the mempool size, `max_txs_bytes`, with
+default 1GB, and includes many smaller improvements and bug-fixes.
+
+See the [v0.31.0
```

Comment:
Nit: this line-break looks not intended. Shows up OK though: https://github.com/tendermint/tendermint/blob/551b6322f5a74f578a8487001e805e4e1da6394d/CHANGELOG.md